libr/anal/fcn.c

```diff
@@ -1516,9 +1516,13 @@ R_API int r_anal_fcn_cc(RAnal *anal, RAnalFunction *fcn) {
 
 R_API char *r_anal_function_get_signature(RAnalFunction *function) {
 	RAnal *a = function->anal;
+	char *fcnname = (r_str_startswith (function->name, "sym.imp."))
```

Comment:
I think we have another way to check for i. ported functions. There's a property telling you whether it's an import. I'm afk, CC @thestr4ng3r probably remember

Reply:
We have that for symbols, but I couldn't find anything like that but for functions. There's another API function, `r_anal_fcn_format_sig`, that parses it in a similar fashion.

Reply:
can an imported function not be a symbol ? I think we made all imported functions symbols in the realname pr.

can't guarantee though

Reply:
this logic is not working everywhere, it was half implemented. thats why this is broken and why this pr

Reply:
Let me request a review from @thestr4ng3r , I remember that we tried to avoid comparing to a substring prefix.  Maybe this can be changed in another way

Reply:
Yes, the way is to get the flag for the function, then check if it is in the imports flagspace.

Reply:
can you point to file:line? or code snippet?